kanban/api/views/issue.py

```diff
@@ -0,0 +1,67 @@
+from kanban.api.serializers import IssueSerializer
+from kanban.models.issue import Issue
+from kanban.tasks import (
+    issue_assignee_change_notification,
+    issue_due_time_notification,
+)
+from rest_framework import viewsets, generics
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from user_account.api.serializers import AssignUserSerializer
+from user_account.models.user_account import UserAccount
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+
+
+class IssueViewSet(viewsets.ModelViewSet):
+    queryset = Issue.objects.select_related("assignee", "project")
+    serializer_class = IssueSerializer
+
+    def perform_update(self, serializer):
+        previous_due_date = serializer.instance.due_date
+        instance = serializer.save()
+
+        if instance.due_date != previous_due_date and instance.assignee is not None:
```

Comment:
What if update changes previously estabilished date or assignee? won't multiple notifications be sent?

Reply:
also to follow srp this could be extracted to a separate function